src/main.c

```diff
@@ -1654,10 +1654,10 @@ static int on_config_listen(h2o_configurator_command_t *cmd, h2o_configurator_co
                 listener = add_listener(fd, ai->ai_addr, ai->ai_addrlen, ctx->hostconf == NULL, 0);
                 listener->quic.ctx = quic;
                 if (quic_node != NULL) {
-                    yoml_t **retry_node, **sndbuf, **rcvbuf, **amp_limit, **qpack_encoder_table_capacity;
+                    yoml_t **retry_node, **sndbuf, **rcvbuf, **amp_limit, **qpack_encoder_table_capacity, **max_stream_data_bidi;
```

Comment:
I think you meant `max_streams_bidi`?

"max_stream_data" and "max_streams" are different things in QUIC (see section [19.10](https://quicwg.org/base-drafts/draft-ietf-quic-transport.html#name-max_stream_data-frames) and 19.11 of QUIC transport draft).

Reply:
I did, thank you!